tgui/packages/tgui/interfaces/CentComViewer.js

```diff
@@ -121,7 +121,16 @@ export const CentComViewer = (props, context) => {
         <Stack fill vertical>
           <Stack.Item>
             <Section
-              title={`Ban Data for ${key}`}
+              title={
+                <>
+                  {`Ban Data for `}
+                  <Input
+                    width="50vw"
+                    value={key}
+                    onChange={(e, value) => act('updateKey', { value })}
```

Comment:
This will result in a change every time a character is typed, right? Could be a bit funky with the API calls. Consider a submit button?

Reply:
No that's the behavior of OnInput, OnChange is only on submission
```markdown
- `onChange: (e, value) => void` - An event, which fires when you commit
the text by either unfocusing the input box, or by pressing the Enter key.
- `onInput: (e, value) => void` - An event, which fires on every keypress.
 ```